### DISCUSSION TITLE
Text Tray Icon for Sensors with Percent Values

### DISCUSSION THREAD

**Discussion by mepherion** [LibreHardwareMonitor/LibreHardwareMonitor#1181] [Ideas]
For sensor values that are displayed as percentage, they are showed as a bar graph in the tray (e.g., CPU Load, Memory Load, etc.)

It would be nice to have an option to change it to represent it as pure text (e.g., "47%")

_See also https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/issues/1170 which was closed_

**Comment by Eta-Beta** [comment]
Good morning, I agree with you, a bar does not give an effective idea of the free or occupied space of an HDD.

**Comment by Nostrildumbass** [comment]
Hate to bump an old post but also hate to open another post for the same subject. Would very much appreciate this option for the gadgets as well; they all seem to show a bar instead of a numerical % value.

**Comment by nykoo** [comment]
same for gadget will be way better. Choice between graph bar or % will be perfect.

**Comment by antarek** [comment]
Same request. I don't want to display a bar graph in my taskbar; it's inconvenient. I need to be able to view the numerical value without having to hover over the bar graph.
